### DISCUSSION TITLE
How do I make something like this link?

### DISCUSSION THREAD

**Discussion by plesnerd** [UseInterstellar/Interstellar#514] [Q&A] [answered]
roblox-interstellar.vercel.app. I want it to open a different link instead of roblox. (for my game site)

**Comment by crllect** [comment, marked as answer]
That link is just an instance of interstellar hosted on vercel, but accessing the landing page redirects to the uv encoded url of roblox on now.gg

To make your own, your gonna need some basics of webdev.

Interstellar isn't built with react or a similar framework, so adding this will be (relatively) simple.
Make an index.html page (replace the one in the fork), that loads all of the uv assets,
then just redirects to the uv encoded url of the site you want to open.

If that didn't make any sense, dont worry, just dm on discord @crllect and ill help you get this sorted
